doc/Customization-guide/Documentation-contribution-and-style-guide/index.md

```diff
@@ -211,19 +213,29 @@ Eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dic
 
 # Documentation contribution
 
-Kaa documentation is a part of [Kaa source code](https://github.com/kaaproject/kaa) - it can be found in the `doc/` folder. A full description of contributing to Kaa project can be found at [How to contribute]({{root_url}}Customization-guide/How-to-contribute/). Key differences between contributing source code and documentation are :
+Kaa documentation is a part of [Kaa source code](https://github.com/kaaproject/kaa) - it can be found in the `doc/` folder.
```

Comment:
Maybe make the "doc folder" a link to
https://github.com/kaaproject/kaa/tree/master/doc
? (It's not in master yet, but will be once we release).